.gitattributes

```diff
@@ -0,0 +1 @@
+yarn.lock binary
```

Comment:
I though this would do the trick (this is what mobile does), but maybe I need to do:
```
yarn.lock linguist-generated
```
This is what webapp does.